### PR TITLE
[SERV-345] Add script for importing items

### PR DESCRIPTION
**Pull request by markmatney**
Will mark as ready for review once the API endpoint exists.

### CODE REVIEW

**Review by ksclarke — DISMISSED** [review]

**Review by DRickard — DISMISSED** [review]

File: src/main/scripts/import-items/import-items.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+class AccessMode(Enum):
+
+    """The access mode choices."""
+    OPEN = 0
+    TIERED = 1
+    ALL_OR_NOTHING = 2
+
+@click.command()
+@click.option('--access-mode', '-m', help='The access mode to use for all items in each INPUT_CSV.', required=True,
```
[the remark this answers is not in the document]
Comment (markmatney) [reply src/main/scripts/import-items/import-items.py:20]: Oh -- is this the "Visibility" field?

[the remark this answers is not in the document]
Comment (ksclarke) [reply src/main/scripts/import-items/import-items.py:20]: Probably. It looks like there is a mapping going on in the front-facing app (https://github.com/UCLALibrary/californica/blob/ae833398b3fa3395317d623ab0a14a99b0295856/app/importers/californica_mapper.rb#L161-L173) so I'm guessing we'll need something like that? We'll see.

[the remark this answers is not in the document]
Comment (ksclarke) [reply src/main/scripts/import-items/import-items.py:20]: I think we can get away with only putting that mapping into Fester though (not this script).

**Review by ksclarke — DISMISSED** [review]

**Comment by markmatney** [comment]
Rebased to incorporate changes from #31

**Review by DRickard — APPROVED** [review]

**Review by ksclarke — APPROVED** [review]

**Pull request merged**